get_genotype_vcf.py

```diff
@@ -133,7 +134,11 @@ def remove_chr_from_bim(input_bim: str, output_bim: str, bim_renamed: str):
 @click.option('--chromosomes', help=' e.g., chr22,chrX ')
 @click.option(
     '--relateds-to-drop-path',
-    default='gs://cpg-bioheart-main-analysis/large_cohort/bioheart1-0/relateds_to_drop.ht',
+    default='gs://cpg-tenk10k-main/large_cohort/tenk10k-genome-2-3-eur/relateds_to_drop.ht',
+)
+@click.option(
+    '--qc_pass_variants_path',
```

Comment:
sorry nit pick but can this be `--qc-pass-variant-path` to match the other click arguments?

Reply:
Ooops 😬